plugins/autosuspend.koplugin/main.lua

```diff
@@ -606,6 +606,8 @@ function AutoSuspend:AllowStandbyHandler()
         -- This shouldn't prevent us from actually consuming any pending input events first,
         -- because if we were woken up by user input, those events should already be in the evdev queue...
         UIManager:consumeInputEarlyAfterPM(true)
+    else
+        self:_start_standby()
```

Comment:
I'd possibly guard it behind an `if not self.going_to_suspend` branch to mimic `onLeaveStandby`, but that looks sensible, yeah ;).